static/js/settings_bots.js

```diff
@@ -92,6 +94,12 @@ exports.set_up = function () {
     $('#payload_url_inputbox').hide();
     $('#create_payload_url').val('');
 
+    // Initially service field will be hidden and will be viewed only on selecting
+    // "Embedded bot" type.
+    $('#service_name_list').hide();
+    $('#select_service_name').val('xkcd'); // TODO: This can be changed to a more generic type bot,
+                                           // such as hello bot etc.
```

Comment:
Aahh, this was initially supposed to be the placeholder bot for the dropdown. 
Instead of the empty dropdown list that we see now, it was supposed to have 'xkcd'. 

Then while working on the bots testing, I realized that almost all the bots need to be worked upon. So, for ease of testing this PR, I thought of adding the simplest bots and forgot to amend this line. My bad! I'll fix this right away.